signbank/dictionary/templates/dictionary/import_manual_validation_csv.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'baselayout.html' %}
+{% load stylesheet %}
+{% load bootstrap3 %}
+{% load i18n %}
+{% block bootstrap3_title %}{% blocktrans %}Import Manual Validation CSV{% endblocktrans %} | {% endblock %}
+
+{% block content %}
+    {% if perms.dictionary.import_csv %}
+    <div id="import-csv">
+        {# Translators: #}
+        <h3>{% blocktrans %}Import Manual Validation CSV{% endblocktrans %}:</h3>
+
+        <ul>
+            <li>{% blocktrans %}CSV-file should contain the following columns:{% endblocktrans %}</li>
+            <ul>
+              <li>{% blocktrans %}group{% endblocktrans %}</li>
```

Comment:
**issue**: this text is hard-coded into the importer and therefore shouldn't be translated